src/kbmod/search/stamp_creator.cpp

```diff
@@ -22,9 +24,9 @@ std::vector<RawImage> StampCreator::create_stamps(ImageStack& stack, const Traje
         if (use_all_stamps || use_index[i]) {
             // Calculate the trajectory position.
             float time = stack.get_zeroed_time(i);
-            PixelPos pos = {trj.x + time * trj.vx, trj.y + time * trj.vy};
+            Point pos{trj.y + time * trj.vy, trj.x + time * trj.vx};
```

Comment:
I don't know. I'm not sure I follow, Can you come up with a test-case or something because at the moment they seem to be passing? This was flipped back to `xy` convention for `Point`.

Reply:
Never mind. This is fine. I was confused because `Point` is (y, x) and `PixelPos` is (x, y)